channeldb/channel.go

```diff
@@ -2196,6 +2188,42 @@ func (c *OpenChannel) AdvanceCommitChainTail(fwdPkg *FwdPkg) error {
 			return err
 		}
 
+		// Persist the unsigned acked updates that are not included
+		// in their new commitment.
+		updateBytes := chanBucket.Get(unsignedAckedUpdatesKey)
+		if updateBytes == nil {
```

Comment:
Could add a comment that if we don't need to filter out any updates, then we return as is since we don't need any further processing.

Reply:
done